### ISSUE TITLE
RDF not giving gold on completing an instance

### ISSUE THREAD

**Issue by AlexCSCP** [dalaranwow/dalaran-wow#5504]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When you finish a instance using the RDF, you get nothing.
In the RDF panel you can see "Money: 7g 40s Experience: 22050" and when you complete the instance you see in the chat "Experience gained: 44100. Received 22 Gold, 20 silver." but you gain no gold.
**Expected behaviour**: Tell us what should happen instead
One should gain some gold when using the random option in the dungeon finder.
**Steps to reproduce the problem**:

1. Open the RDF panel, select heal, hit "Find group" and wait 4 hours for a tank.
2. Run behind the tank healing their ass.
3. After the last boss, see the "Dungeon complete!" message and look at your bag to see that you didn't get any gold.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
Before killing the last boss:
https://imgur.com/4omXbiT.png
After killing the last boss:
https://imgur.com/but1DYb.png
You can see the "Dungeon complete!" message and my bag with the same amount of gold.
